service/ci/publish_to_github.sh

```diff
@@ -1,17 +1,21 @@
 # load .env defines in root of repo
 export $(egrep -v '^#' .env | xargs)
-export GITHUB_WEB_REGISTRY_URI=ghcr.io/NREL/boptest-web
-export GITHUB_WORKER_REGISTRY_URI=ghcr.io/NREL/boptest-worker
+export WEB_REGISTRY_URI=boptest_service-web
+export WORKER_REGISTRY_URI=boptest_service-worker
+export GITHUB_ORG=tijcolem
```

Comment:
Double checking, should GITHUB_ORG be tijcolem or ibpsa, given step 2 listed in this [comment](https://github.com/ibpsa/project1-boptest/pull/722#issue-2781114341)?

Reply:
Thanks for catching this!  That was from my testing on my fork. I changed to ibpsa.